tracy-client/src/lib.rs

```diff
@@ -459,37 +574,39 @@ const fn adjust_stack_depth(depth: u16) -> u16 {
 mod tests {
     use super::*;
 
-    #[global_allocator]
-    static GLOBAL: ProfiledAllocator<alloc::System> = ProfiledAllocator::new(alloc::System, 100);
```

Comment:
This in particular stopped working with the `libtest` harness because `libtest` creates some allocations before we get an opportunity to enable the client, and so some allocations get lost, and tracy is not very happy about it.

Reply:
Could `ProfiledAllocator` itself call `enable` if necessary?

Reply:
Well, there are two options I think if we wanted to make this transparent to the users: go back to life-before-main and `enable` when `ProfiledAllocator` is “constructed”. Or alternatively we could call `enable`/`disable` for each allocation/deallocation, at a cost of some extra overhead for each allocation.

I think either option is pretty subpar.